### DISCUSSION TITLE
add history icon

### DISCUSSION THREAD

**Discussion by yabasha** [tailwindlabs/heroicons#474] [Ideas]
Hi,

I was searching for history icon in https://heroicons.dev/ but I couldn't find it, so I made one so add it to the list for anyone searching for similar one

add history icon to the list:

<svg class="h-6 w-6 text-gray-400 group-hover:text-gray-500 group-focus:text-gray-500 transition ease-in-out duration-150"
         stroke="currentColor"
         fill="none"
         viewBox="0 0 24 24">
        <path d="M13,3a9,9,0,0,0-9,9H1l3.89,3.89L5,16l4-4H6a7,7,0,1,1,2.06,4.94L6.64,18.36A9,9,0,1,0,13,3ZM12,8v5l4.28,2.54L17,14.33l-3
        .5-2.08V8Z"/>
    </svg>

;)

**Comment by ndri** [comment]
I added `arrow-path-clock` to [Sidekickicons](https://github.com/ndri/sidekickicons/), which are meant to be used as a companion to Heroicons. Browse them here: https://sidekickicons.com/?icon=arrow-path-clock

**Reply by fabswt** [reply]
Nice site, thank you! Maybe using a UI closer to that of the original Heroicons site (white background, square, name of icon below the icon container) would make sense?

Just my $0.02.

**Reply by ndri** [reply]
Thanks for the feedback! I didn't want to rip off the original site too much 😄 but you can switch to light mode if you prefer the white background.
